src/app/register/logout/get.controller.js

```diff
@@ -4,6 +4,6 @@ module.exports = (req, res) => {
   const cookie = new CookieModel(req);
   req.session.destroy(() => {
     cookie.reset();
-    res.redirect('/login');
+    res.redirect('/welcome/index');
```

Comment:
When logging out, just goes to the home page instead of the login page. Matter of preference really?